tests/sample/cgroup_sock_addr.c

```diff
@@ -34,12 +34,9 @@ authorize_v4(bpf_sock_addr_t* ctx, struct bpf_map_def* connection_policy_map)
     connection_tuple_t tuple_key = {0};
     int* verdict = NULL;
 
-    tuple_key.src_ip.ipv4 = ctx->msg_src_ip4;
```

Comment:
what was the need for changing this?

Reply:
Now the `ctx` passsed to `cgroup/connect*` will not contain the source information. So we cannot use that as key.